map/styled_point.cpp

```diff
@@ -0,0 +1,54 @@
+#include "map/styled_point.hpp"
+
+#include "base/stl_add.hpp"
+#include "base/string_utils.hpp"
+
+#include "std/fstream.hpp"
+#include "std/algorithm.hpp"
+#include "std/auto_ptr.hpp"
+
+
+graphics::DisplayList * StyledPoint::GetDisplayList(UserMarkDLCache * cache) const
+{
+  return cache->FindUserMark(UserMarkDLCache::Key(GetStyle(), graphics::EPosAbove, GetContainer()->GetDepth()));
```

Comment:
Original API style is retired by this PR. I think all points are drawn in correct location, as shown above. Unstyled points are drawn as placemark-red now. So I think this code is correct.